action.yml

```diff
@@ -17,8 +17,8 @@ inputs:
     description: 'Also look for tags to delete'
     required: false
     default: false
-  include_prefixes:
-    description: 'List of branch prefixes to be deleted'
+  extra_protected_branch_regex:
```

Comment:
@hwoarang what about providing a default value for the extra protected branches to include common default branches names, like `main` and `master`? That way, users don't have to add such branch names. What do you think?